### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by the-andrew-007**

### CODE REVIEW

File: taxi/urls.py [hunk 1]
```diff
@@ -22,6 +28,24 @@
     path(
         "drivers/<int:pk>/", DriverDetailView.as_view(), name="driver-detail"
     ),
+    path("car/create", CreateCarView.as_view(), name="car-create"),
```
Comment (vasylhnatiuk) [inline taxi/urls.py:31]:
```suggestion
    path("car/create/", CreateCarView.as_view(), name="car-create"),
```

File: taxi/urls.py [hunk 2]
```diff
@@ -22,6 +28,24 @@
...
+    path("car/<int:pk>/update", UpdateCarView.as_view(), name="car-update"),
```
Comment (vasylhnatiuk) [inline taxi/urls.py:32]:
```suggestion
    path("car/<int:pk>/update/", UpdateCarView.as_view(), name="car-update"),
```

File: taxi/urls.py [hunk 3]
```diff
@@ -22,6 +28,24 @@
...
+    path("car/<int:pk>/delete", DeleteCarView.as_view(), name="car-delete"),
```
Comment (vasylhnatiuk) [inline taxi/urls.py:33]:
```suggestion
    path("car/<int:pk>/delete/", DeleteCarView.as_view(), name="car-delete"),
```

Comment (vasylhnatiuk) [reply taxi/urls.py:33]: add here and in all of path

**Review by vasylhnatiuk — CHANGES_REQUESTED** [review]

**Review by ArsenPidhoretskyi — APPROVED** [review]
Well done!

**Pull request closed without merging**